util/GO9PTERM

```diff
@@ -16,8 +16,11 @@ if [ "$(uname)" = "Linux" ] && [ -e /dev/kvm ]; then
 	fi
 fi
 
+# specific usb devices added by '-device usb-host,hostbus=X,hostaddr=X'
+# replacing 'X' with the device numbers provided by a tool like 'lsusb'
 read -r cmd <<EOF
 $kvmdo qemu-system-x86_64 -s -cpu Opteron_G1 -smp 2 -m 2048 $kvmflag \
+-usb \
```

Comment:
ok, how about you show the result of executing those commands and then what you add to the qemu command?

I'll try this PR now.